src/shared/processor-templates/otoken/otoken.ts

```diff
@@ -28,6 +28,7 @@ import { Context } from '../../../processor'
 import { type Transaction, activityFromTx } from '../../../utils/activityFromTx'
 import { ADDRESS_ZERO } from '../../../utils/addresses'
 import { DECIMALS_18 } from '../../../utils/constants'
+import { multicall } from '../../../utils/multicall'
```

Comment:
viem has a built in multicall btw <https://viem.sh/docs/contract/multicall.html>

Reply:
I am using a multicall contract in most cases, however when we are prior to that contract's deploy date I make an rpc batch call instead. It does result in less back and forth which improves performance still. This function gives us a single interface no matter what block height we're at.